src/Maybe.test.ts

```diff
@@ -9,8 +11,9 @@ test("Maybe is requireable", (t: TestAssertions) => {
 });
 
 test("Making a maybe value gives a Some", (t: TestAssertions) => {
-  t.ok(Maybe.some(10));
-  t.same(Maybe.some(10), new Maybe.Some(10));
+  jsc.assert(jsc.forall("nat", (i: number) => {
+    return deepEqual(Maybe.some(i), new Maybe.Some(i));
+  }));
   t.end();
```

Comment:
@phuu Btw, upgrade to latest version of AVA and you can drop all the `t.end()` ;)